frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,182 @@
+import { FluentVariable } from "@fluent/bundle";
+import { Localized, useLocalization } from "@fluent/react";
+import {
+  getBundlePrice,
+  getBundleSubscribeLink,
+  isBundleAvailableInCountry,
+} from "../../functions/getPlan";
+import { isFlagActive } from "../../functions/waffle";
+import { RuntimeData } from "../../hooks/api/runtimeData";
+import { MozillaVpnWordmark } from "../Icons";
+import styles from "./BundleBanner.module.scss";
+import { LinkButton } from "../Button";
+import womanInBanner400w from "./images/bundle-banner-woman-400w.png";
+import womanInBanner768w from "./images/bundle-banner-woman-768w.png";
+import womanInBanner1280w from "./images/bundle-banner-woman-1280w.png";
+import bundleFloatOne from "./images/bundle-float-1.svg";
+import bundleFloatTwo from "./images/bundle-float-2.svg";
+import bundleFloatThree from "./images/bundle-float-3.svg";
+import bundleLogo from "./images/vpn-and-relay-logo.svg";
+import { trackPurchaseStart } from "../../functions/trackPurchase";
+
+export type Props = {
+  runtimeData: RuntimeData;
+};
+
+type FloatingFeaturesProps = {
+  icon: string;
+  text: string;
+  position: string;
+  vars?: Record<string, FluentVariable>;
+};
+
+const FloatingFeatures = (props: FloatingFeaturesProps) => {
+  const { l10n } = useLocalization();
+
+  const text = props.vars ? (
+    <Localized id={props.text} vars={props.vars}>
+      <span className={styles["float-features-text"]} />
+    </Localized>
+  ) : (
+    <span className={styles["float-features-text"]}>
+      {l10n.getString(props.text)}
+    </span>
+  );
+
+  return (
+    <div
+      className={`${styles[props.position]} ${styles["float-features-item"]}`}
+    >
+      <img alt="" src={props.icon} />
+      {text}
+    </div>
+  );
+};
+
+export const BundleBanner = (props: Props) => {
+  const { l10n } = useLocalization();
+
+  const mainImage = (
+    <img
+      src={womanInBanner400w.src}
+      srcSet={`${womanInBanner400w.src} 400w, ${womanInBanner768w.src} 768w, ${womanInBanner1280w.src} 1280w`}
+      alt=""
+      className={styles["main-image"]}
+    />
+  );
+
+  return (
+    <div className={styles["bundle-banner-wrapper"]}>
```

Comment:
@codemist should this whole component be wrapped around a flag check?

Reply:
The flag is wrapped in `index.page.tsx` and `premium.page.tsx`